src/pages/events.vue

```diff
@@ -48,6 +48,7 @@ const groupedEventData = computed(() => {
     const events: Record<number, VatsimEvent[]> = {};
 
     data.value?.events.forEach(event => {
+        if (new Date(event.end_time) < new Date('2025-01-10T20:00:01Z')) return;
```

Comment:
```suggestion
        if (new Date(event.end_time) < new Date()) return;
```

Also I'm sure TypeScript will require .getTime() for this to work properly

Reply:
In TypeScript / JavaScript, Date objects can be compared directly. There is no need to use .getTime() here.